entity-framework/core/modeling/generated-properties.md

```diff
@@ -67,15 +67,13 @@ Similarly, a property can be configured to have its value generated on add or up
 
 ***
 
-> [!WARNING]
-> Unlike with default values or computed columns, we are not specifying *how* the values are to be generated; that depends on the database provider being used. Database providers may automatically set up value generation for some property types, but others may require you to manually set up how the value is generated.
->
-> For example, on SQL Server, when a GUID property is configured as value generated on add, the provider automatically performs value generation client-side, using an algorithm to generate optimal sequential GUID values. However, specifying <xref:Microsoft.EntityFrameworkCore.Metadata.Builders.PropertyBuilder.ValueGeneratedOnAdd%2A> on a DateTime property will have no effect ([see the section below for DateTime value generation](#datetime-value-generation)).
->
-> Similarly, byte[] properties that are configured as generated on add or update and marked as concurrency tokens are set up with the rowversion data type, so that values are automatically generated in the database. However, specifying <xref:Microsoft.EntityFrameworkCore.Metadata.Builders.PropertyBuilder.ValueGeneratedOnAdd%2A> has no effect.
->
-> [!NOTE]
-> Depending on the database provider being used, values may be generated client side by EF or in the database. If the value is generated by the database, then EF may assign a temporary value when you add the entity to the context; this temporary value will then be replaced by the database generated value during `SaveChanges()`. For more information, [see the docs on temporary values](xref:core/change-tracking/explicit-tracking#temporary-values).
```

Comment:
This seems like under-the-hood implementation details that shouldn't necessarily interest the user (not on this introductory page, in any case). But if we think it's important for everyone to be exposed to this, I can bring it back.